app/sync.py

```diff
@@ -29,10 +30,10 @@ def getinfo():
     with profile_session_scope() as session:
         profile = Profile.get_active(session)
         try:
-            result = client.getinfo()['result']
-            signals.getinfo.emit(Getinfo(**result))
-            if result['balance'] != profile.balance:
-                profile.balance = result['balance']
+            info = client.getinfo()
+            signals.getinfo.emit(Getinfo(**info))
```

Comment:
You might want to use the `Getinfo` object from the new `mcrpc.responses` module here.  Even better, we could quite easily teach mcrpc to automatically return python objects for the auto-generated response types that we support. See: https://github.com/coblo/mcrpc/blob/master/mcrpc/responses.py

Reply:
Wait... `mcrpc` should automaticaly return a `Getinfo` object: https://github.com/coblo/mcrpc/blob/master/mcrpc/base.py#L70
Did you install and are you using latest version 0.2.3 of `mcrpc`?